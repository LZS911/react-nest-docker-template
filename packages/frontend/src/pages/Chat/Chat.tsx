import ButtonDemo from '../../components/ThemeBase/Button/Demo/Button';

const Chat: React.FC = () => {
  return (
    <div>
      <ButtonDemo />
    </div>
  );
};
export default Chat;
