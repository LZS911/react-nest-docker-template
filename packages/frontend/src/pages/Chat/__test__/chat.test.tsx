import Chat from '..';
import { renderWithRedux } from '../../../testUtils/customRender';
describe('test Chat', () => {
  it('should render Chat to match snapshot', () => {
    const { container } = renderWithRedux(<Chat />);
    expect(container).toMatchSnapshot();
  });
});
