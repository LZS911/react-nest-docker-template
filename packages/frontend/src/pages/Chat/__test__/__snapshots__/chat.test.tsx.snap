// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Chat should render Chat to match snapshot 1`] = `
<div>
  <div>
    <div>
      <button
        class="ant-btn ant-btn-link ant-btn-dangerous"
        type="button"
      >
        <span
          class=""
        >
          changeThemeMode
        </span>
      </button>
      <button
        class="ant-btn ant-btn-primary  border-primary focus:border-primary hover:border-primary !bg-primary"
        type="button"
      >
        <span
          class="!text-white"
        >
          blue
        </span>
      </button>
      <button
        class="ant-btn ant-btn-default  border-secondary focus:border-secondary hover:border-secondary !bg-secondary"
        type="button"
      >
        <span
          class="!text-black dark:!text-white"
        >
          purple
        </span>
      </button>
      <button
        class="ant-btn ant-btn-default  dark:bg-darkMode focus:border-primary hover:border-primary focus:!text-primary hover:!text-primary"
        type="button"
      >
        <span
          class="dark:text-white hover:!text-primary focus:!text-primary"
        >
          green
        </span>
      </button>
      <button
        class="ant-btn ant-btn-dashed  dark:bg-darkMode focus:border-primary hover:border-primary focus:!text-primary hover:!text-primary"
        type="button"
      >
        <span
          class="dark:text-white hover:!text-primary focus:!text-primary"
        >
          dashed
        </span>
      </button>
      <button
        class="ant-btn ant-btn-ghost  dark:bg-darkMode focus:border-primary hover:border-primary focus:!text-primary hover:!text-primary"
        type="button"
      >
        <span
          class="dark:text-white hover:!text-primary focus:!text-primary"
        >
          ghost
        </span>
      </button>
      <button
        class="ant-btn ant-btn-default  dark:bg-darkMode focus:border-primary hover:border-primary focus:!text-primary hover:!text-primary"
        type="button"
      >
        <span
          class="dark:text-white hover:!text-primary focus:!text-primary"
        >
          default
        </span>
      </button>
      <button
        class="ant-btn ant-btn-default ant-btn-dangerous  dark:bg-darkMode "
        type="button"
      >
        <span>
          danger
        </span>
      </button>
    </div>
  </div>
</div>
`;
